### DISCUSSION TITLE
Question about building from source code

### DISCUSSION THREAD

**Discussion by zengkaipeng** [rdkit/rdkit#8032] [Q&A]
I am trying to build a customized rdkit-python for myself, but I am stuck at the step of compiling from the C++ source code. In fact, I haven't made any modifications to the code yet. I followed the instructions on https://github.com/rdkit/rdkit/blob/Release_2024_09/Docs/Book/Install.md for compilation, the only difference being that I used `/usr/bin/cmake` for compilation instead of the one installed by Anaconda. Then, when I ran ctest, I encountered the error shown in the attached image. Is this because I didn't use the cmake installed by Anaconda for compilation, or is it that the original compilation command requires specifying additional variables? (I can find libboost_iostreams.so.1.82.0 in $CONDA_PREFIX/lib, but ctest reported an error.)

![1732386426794](https://github.com/user-attachments/assets/e4773833-71ac-4532-a21b-aa58a1920384)

**Comment by greglandrum** [comment]
The first of those errors is telling you that your LD_LIBRARY_PATH is set incorrectly. The second one may be pointing to the same thing.
If you installed boost with conda, then you probably need to make sure that both `$RDBASE/lib` and `$CONDA_PREFIX/lib` are in LD_LIBRARY_PATH

**Reply by zengkaipeng** [reply]
After my attempts, I discovered that the first error occurred because I used `/usr/lib/cmake` to compile the project. Now all the errors are shown in the image below.

![1732429929144](https://github.com/user-attachments/assets/003d8b9b-59c2-404d-bc2c-055c1d43eb2d)


Regarding the GLIBC errors, we can find information in the installation instructions from [this link](https://github.com/rdkit/rdkit/blob/Release_2024_09/Docs/Book/Install.md), but the problem is that even though I have installed gxx_linux-64, I am still encountering this error.

![1732430442294](https://github.com/user-attachments/assets/201c0672-8601-4750-824d-33b05de646ed)


Then how should I solve the problem above？ Thanks in advance.
